openapi/data/fields.py

```diff
@@ -349,7 +354,7 @@ def __call__(self, field, value, data=None):
 class DecimalValidator(NumberValidator):
     def __call__(self, field, value, data=None):
         try:
-            value = decimal.Decimal(value)
```

Comment:
what's the point of this?

Reply:
Isn't this done by our custom json `loads`? As soon as we build a float it's game over for ensuring precision.